app/src/main/java/com/android/sample/ui/activitydetails/ActivityDetailsScreen.kt

```diff
@@ -0,0 +1,171 @@
+
+          //  price and schedule
+          Spacer(modifier = Modifier.height(8.dp))
+          Row(
+              verticalAlignment = Alignment.CenterVertically,
+              horizontalArrangement = Arrangement.SpaceBetween,
+              modifier = Modifier.fillMaxWidth()) {
+                Row(verticalAlignment = Alignment.CenterVertically) {
+                  Icon(Icons.Filled.AttachMoney, contentDescription = "Price")
+                  Spacer(modifier = Modifier.width(4.dp))
+                  Text(text = if (price != null) "${price.toString()} CHF" else "not defined yet")
+                }
+              }
+          Spacer(modifier = Modifier.height(8.dp))
+          Row(verticalAlignment = Alignment.CenterVertically) {
+            Icon(Icons.Default.DateRange, contentDescription = "Schedule")
+            Spacer(modifier = Modifier.width(4.dp))
+            Text(text = schedule ?: "not defined yet")
+          }
+
+          Spacer(modifier = Modifier.height(32.dp))
+
+          // Enroll button
+          Button(
+              onClick = {
+                if ((placesLeft ?: 0) > 0) {
+                  val theActivity =
+                      Activity(
+                          uid = activityRepositoryFirestore.getNewUid(),
+                          title = activityTitle ?: "",
+                          description = description ?: "",
+                          date = schedule ?: "",
+                          price = price ?: 0.0,
+                          placesLeft = (placesLeft ?: 0).let { if (it > 0) it - 1 else 0 },
+                          maxPlaces = maxPlaces ?: 0,
+                          creator = creator ?: "",
+                          status = status ?: ActivityStatus.ACTIVE,
+                          location = location ?: "",
+                          images = listOf(),
+                      )
+                  listActivityViewModel.addActivity(theActivity)
+                  Toast.makeText(context, "Enroll Successful", Toast.LENGTH_SHORT).show()
+                  navigationActions.navigateTo(Screen.OVERVIEW)
+                } else {
+                  Toast.makeText(
+                          context, "Enroll failed, limit of places reached", Toast.LENGTH_SHORT)
+                      .show()
+                }
+              },
+              modifier = Modifier.fillMaxWidth().padding(horizontal = 24.dp)) {
+                Text(text = "Enroll")
+              }
+        }
+      }
+}
+
+@Composable
+fun Imagery() {
+  LazyRow(
+      modifier = Modifier.fillMaxWidth().height(200.dp).padding(3.dp),
+  ) {
+    items(com.android.sample.ui.activity.items.size) { index ->
+      Image(
+          painter = painterResource(id = com.android.sample.ui.activity.items[index].imageResId),
+          contentDescription = com.android.sample.ui.activity.items[index].contentDescription,
+          contentScale = ContentScale.Crop,
+          modifier = Modifier.padding(8.dp))
+    }
```

Comment:
Good that you started implementing this. I guess we still need to define exactly how we will work with the images. Good base for the next sprint.